src/components/mgt-agenda/mgt-agenda.ts

```diff
@@ -21,47 +21,83 @@ import { MgtPersonDetails } from '../mgt-person/mgt-person';
 @customElement('mgt-agenda')
 export class MgtAgenda extends MgtTemplatedComponent {
   private _firstUpdated = false;
+  /**
+   * _isNarrow property, determines width available for agenda component.
```

Comment:
[Same for all other]

no need to specify the name of the property in the docs. For example, just `determines available width for agenda component` is sufficient.

Reply:
updated  

(not a good reference, but in case):

https://github.com/microsoftgraph/microsoft-graph-toolkit/tree/nivogt/tsdocs-update/src/components